cluster/gce/gci/master-helper.sh

```diff
@@ -37,8 +37,30 @@ function create-master-instance {
 
   write-master-env
   ensure-gci-metadata-files
-  gcloud compute instances create "${MASTER_NAME}" \
-    ${address_opt} \
+  create-master-instance-internal "${MASTER_NAME}" "${address_opt}"
+}
+
+function replicate-master-instance() {
+  local existing_master_zone=${1}
+  local existing_master_name=${2}
+  local existing_master_replicas=${3}
+
+  local kube_env=$(get-metadata ${existing_master_zone} ${existing_master_name} kube-env)
+  # Substitute INITIAL_ETCD_CLUSTER to enable etcd clustering.
+  kube_env=$(echo "${kube_env}" | grep -v "INITIAL_ETCD_CLUSTER")
+  kube_env=$(echo -e "${kube_env}\nINITIAL_ETCD_CLUSTER: '${existing_master_replicas},${REPLICA_NAME}'")
+  echo "${kube_env}" > ${KUBE_TEMP}/master-kube-env.yaml
+  get-metadata ${existing_master_zone} ${existing_master_name} cluster-name > ${KUBE_TEMP}/cluster-name.txt
+  get-metadata ${existing_master_zone} ${existing_master_name} gci-update-strategy > ${KUBE_TEMP}/gci-update.txt
+  get-metadata ${existing_master_zone} ${existing_master_name} gci-ensure-gke-docker > ${KUBE_TEMP}/gci-ensure-gke-docker.txt
+  get-metadata ${existing_master_zone} ${existing_master_name} gci-docker-version > ${KUBE_TEMP}/gci-docker-version.txt
+
+  create-master-instance-internal "${REPLICA_NAME}"
+}
+
+function create-master-instance-internal() {
+  gcloud compute instances create "${1}" \
```

Comment:
nit: can you please introduce local named variables for parameter of this function (similarly to what you did in replicate-master-instance function)? This would make it much more readable.